src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
+    }
+
+    public void stopWork() {
```

Comment:
The method 'stopWork' should be annotated with @Override since it's overriding the method from the superclass Machine. This helps to ensure that you are indeed overriding a method you intend to.